src/components/CalendarDay.js

```diff
@@ -10,7 +10,9 @@ const CalendarDay = ({day, isToday, events}) => {
     });
 
     return (
-        <div className={dayClasses}>
+      <div 
+        onClick={onClick.bind(null, 'day', day)}
```

Comment:
Passing `this` would be useful for making the tooltip show up directly above/beside the day the user clicked on, so if we could do that then personally I think it'd be worthwhile to switch to a `class`.